scripts/upgrade_admin_server.sh

```diff
@@ -91,8 +99,13 @@ n.target_platform = 'suse-12.2'
 n.provisioner.default_os = 'suse-12.2'
 n.save"
 
-    # Signalize that the upgrade correctly ended
-    echo "12.2" >> $UPGRADEDIR/admin-server-upgraded-ok
+    ret=$?
+    if [ $ret != 0 ]; then
+        report_failure($ret, "Setting the platform to suse-12.2 has failed.")
```

Comment:
Yes, I was thinking about that, and I'm not sure. It may actually make sense to run the rest of the script anyway, and user could try to just fix these os values on the upgraded server.

I'm not sure if failing on Cloud6 side makes sense, as the crowbar has been already upgraded - I think restarting the whole step will not be possible. (This is related also to the other possible failure actually)